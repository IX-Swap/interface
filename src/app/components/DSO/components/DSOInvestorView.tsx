import React, { Fragment, useState } from 'react'
import { DSOInvestorViewHeader } from 'app/components/DSO/components/DSOInvestorViewHeader'
import { DigitalSecurityOffering } from 'types/dso'
import { Tabs, Tab, Grid } from '@mui/material'
import { DSOPricingViewCompact } from 'app/components/DSO/components/DSOPricingViewCompact'
import { DSOTermsViewCompact } from 'app/components/DSO/DSOPreview/DSOTermsViewCompact'
import { TabPanel } from 'components/TabPanel'
import { DSOInvestorInformationView } from 'app/components/DSO/components/DSOInvestorInformationView'
import { DSODataroomView } from 'app/components/DSO/components/DSODataroomView'
import { DSOFAQsView } from 'app/components/DSO/components/DSOFAQsView'
import { DSOVideoLinksView } from 'app/components/DSO/components/DSOVideoLinksView'

export interface DSOInvestorViewProps {
  dso: DigitalSecurityOffering
}

export const DSOInvestorView = (props: DSOInvestorViewProps) => {
  const { dso } = props
  const [selectedIdx, setSelectedIdx] = useState(0)
  const isDSOVideosVisible = dso.videos !== undefined && dso.videos.length > 0
  const isDSOFAQsVisible = dso.faqs !== undefined && dso.faqs.length > 0

  return (
    <Fragment>
      <DSOInvestorViewHeader dso={dso} />

      <Tabs
        value={selectedIdx}
        onChange={(_, index) => setSelectedIdx(index)}
        indicatorColor='primary'
        textColor='primary'
      >
        <Tab label='Overview' />
        <Tab label='Information' />
        <Tab label='Documents' />
        {isDSOVideosVisible ? <Tab label='Videos' /> : null}
        {isDSOFAQsVisible ? <Tab label='FAQs' /> : null}
      </Tabs>

      <TabPanel value={selectedIdx} index={0}>
        <Grid container spacing={9}>
          <Grid item xs={12}>
            <DSOPricingViewCompact dso={dso} />
          </Grid>

          {dso.isCampaign !== true && (
            <Grid item xs={12}>
              <DSOTermsViewCompact dso={dso} />
            </Grid>
          )}
        </Grid>
      </TabPanel>

      <TabPanel value={selectedIdx} index={1}>
        <DSOInvestorInformationView dso={dso} />
      </TabPanel>

      <TabPanel value={selectedIdx} index={2}>
        <DSODataroomView dso={dso} showTitle={false} />
      </TabPanel>

      {isDSOVideosVisible ? (
        <TabPanel value={selectedIdx} index={3}>
          <DSOVideoLinksView dso={dso} />
        </TabPanel>
      ) : null}

      {isDSOFAQsVisible ? (
        <TabPanel value={selectedIdx} index={4}>
          <DSOFAQsView dso={dso} isTitleVisible isNewThemeOn />
        </TabPanel>
      ) : null}
    </Fragment>
  )
}
