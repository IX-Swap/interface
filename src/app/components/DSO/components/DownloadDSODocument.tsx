import React from 'react'
import { useDownloadRawFile } from 'hooks/useDownloadRawFile'
import { IconButton, Tooltip } from '@mui/material'
import { convertBlobToFile, openFileInNewTab } from 'hooks/utils'
import { Launch } from '@mui/icons-material'

export interface DownloadDSODocumentProps {
  dsoId: string
  documentId: string
  type?: 'document' | 'subscriptionDocument'
}

export const DownloadDSODocument = (props: DownloadDSODocumentProps) => {
  const { dsoId, documentId, type = 'document' } = props
  const uri =
    type === 'document'
      ? `/issuance/dso/dataroom/documents/raw/${dsoId}/${documentId}`
      : `/issuance/dso/dataroom/subscription/raw/${dsoId}`
  const [download, { isLoading }] = useDownloadRawFile(uri, {
    onSuccess: ({ data }) => {
      const file = convertBlobToFile(data, '')
      openFileInNewTab(file)
    }
  })
  const handleDownload = async () => await download()

  return (
    <Tooltip title='Download File'>
      <IconButton disabled={isLoading} onClick={handleDownload} size='large'>
        <Launch color='disabled' style={{ width: 23, height: 23 }} />
      </IconButton>
    </Tooltip>
  )
}
