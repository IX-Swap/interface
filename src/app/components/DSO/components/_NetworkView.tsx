import React from 'react'
import { useAllNetworks } from 'app/pages/accounts/pages/withdrawalAddresses/hooks/useAllNetworks'

export interface NetworkViewProps {
  networkId: string
}

export const NetworkView = (props: NetworkViewProps) => {
  const { networkId } = props
  const { data, isLoading } = useAllNetworks()
  const network = data?.find(network => network._id === networkId)

  return isLoading ? <>'...' </> : <>{network?.name}</>
}
