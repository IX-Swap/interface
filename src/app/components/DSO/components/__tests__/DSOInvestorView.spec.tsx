import { DSOInvestorView } from 'app/components/DSO/components/DSOInvestorView'
import React from 'react'
import { render, cleanup } from 'test-utils'
import { dso } from '__fixtures__/authorizer'

jest.mock(
  'app/pages/invest/components/DSOBlockChainDetails/DSOBlockchainDetails',
  () => ({
    DSOBlockchainDetails: jest.fn(() => null)
  })
)

describe('DSOInvestorView', () => {
  afterEach(async () => {
    await cleanup()
    jest.clearAllMocks()
  })

  it('renders without errors', () => {
    render(<DSOInvestorView dso={dso} />)
  })

  it('matches snapshot', () => {
    const { container } = render(<DSOInvestorView dso={dso} />)
    expect(container).toMatchSnapshot()
  })
})
