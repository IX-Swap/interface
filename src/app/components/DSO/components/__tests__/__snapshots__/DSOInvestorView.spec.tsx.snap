// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DSOInvestorView matches snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1a72twe-MuiPaper-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-3 MuiGrid-grid-xs-12 css-1kko4jv-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-3 MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-rwj4a4-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item css-b97a1x-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-ofrcno-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                  >
                    <div
                      class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-69x3yt-MuiAvatar-root"
                      style="width: 60px; height: 60px;"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-1ql7le6-MuiSvgIcon-root-MuiAvatar-fallback"
                        data-testid="PersonIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                        />
                      </svg>
                    </div>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                  >
                    <h4
                      class="MuiTypography-root MuiTypography-h4 css-vxrdkt-MuiTypography-root"
                    >
                      token name
                    </h4>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                    >
                      issuer name
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item css-l7ms5s-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-a5rdam-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                    >
                      Unit Price
                    </h6>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
                    >
                      1 SGD
                    </h6>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-3 MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-bqdoyz-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-8 css-18jsq1j-MuiGrid-root"
              />
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-13lqwx6-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-10zunq2"
                >
                  <a
                    class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-disableElevation MuiButtonBase-root makeStyles-button-15 css-1b0mpvw-MuiTypography-root-MuiLink-root-MuiButtonBase-root-MuiButton-root"
                    href="/app/invest/offerings/5f3152ef2d2dfb4291fccce2/1/view/make-investment"
                    tabindex="0"
                  >
                    Invest
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
          >
            <div
              class="ForwardRef(Box)-root-22 ForwardRef(Box)-root-23 MuiBox-root css-0"
            />
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-144uxk9-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-a5rdam-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                    >
                      Total Units
                    </h6>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
                    >
                      100000 token symbol
                    </h6>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-144uxk9-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-a5rdam-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                    >
                      Total Fundraising Amount
                    </h6>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
                    >
                      100000 SGD
                    </h6>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-144uxk9-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-a5rdam-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                    >
                      Minimum Amount
                    </h6>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
                    >
                      100.00 SGD
                    </h6>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-144uxk9-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-a5rdam-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                    >
                      Minimum Units
                    </h6>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
                    >
                      100.00 token symbol
                    </h6>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-8 css-1vgr1nd-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-8msu0r-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-8atqhb"
        >
          <div
            class="MuiTabs-root css-1ujnqem-MuiTabs-root"
          >
            <div
              class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
              style="overflow: hidden; margin-bottom: 0px;"
            >
              <div
                class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
                role="tablist"
              >
                <button
                  aria-selected="true"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1ccduzc-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="0"
                  type="button"
                >
                  Information
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1ccduzc-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Documents
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1ccduzc-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Videos
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1ccduzc-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  FAQs
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
              <span
                class="MuiTabs-indicator css-50jx75-MuiTabs-indicator"
                style="left: 0px; width: 0px;"
              />
            </div>
          </div>
        </div>
        <div
          class="ForwardRef(Box)-root-22 ForwardRef(Box)-root-24 MuiBox-root css-0"
        />
        <div
          class="MuiBox-root css-12oxtbe"
        >
          <div
            role="tabpanel"
          >
            <div
              class="MuiBox-root css-1161qt5"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                >
                  <div
                    class="MuiBox-root css-0"
                  >
                    <h5
                      class="MuiTypography-root MuiTypography-h5 css-183wt5r-MuiTypography-root"
                    >
                      Company Profile
                    </h5>
                    <div
                      class="MuiBox-root css-1gb8ofc"
                    >
                      <div>
                        introduction
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                >
                  <div
                    class="MuiBox-root css-0"
                  >
                    <h5
                      class="MuiTypography-root MuiTypography-h5 css-183wt5r-MuiTypography-root"
                    >
                      Business Model
                    </h5>
                    <div
                      class="MuiBox-root css-1gb8ofc"
                    >
                      <div>
                        business model
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                >
                  <div
                    class="MuiBox-root css-0"
                  >
                    <h5
                      class="MuiTypography-root MuiTypography-h5 css-183wt5r-MuiTypography-root"
                    >
                      Use of Proceeds
                    </h5>
                    <div
                      class="MuiBox-root css-1gb8ofc"
                    >
                      <div>
                        use of proceeds
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                >
                  <div
                    class="MuiBox-root css-0"
                  >
                    <h5
                      class="MuiTypography-root MuiTypography-h5 css-183wt5r-MuiTypography-root"
                    >
                      Fundraising Milestone
                    </h5>
                    <div
                      class="MuiBox-root css-1gb8ofc"
                    >
                      <div>
                        fundraising milestone
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
                    >
                      <h5
                        class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
                      >
                        Team Members
                      </h5>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            hidden=""
            role="tabpanel"
          />
          <div
            hidden=""
            role="tabpanel"
          />
          <div
            hidden=""
            role="tabpanel"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-3 MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-3mm3v9-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-7ncgri-MuiPaper-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 css-vxrdkt-MuiTypography-root"
              >
                Main Info
              </h4>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Network
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    Ropsten Test Network (Public Ethereum)
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Decimal
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    0
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Capital Structure
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    capital structure
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Launch Date
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    01/01/2000
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Completion Date
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    12/12/2220
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-7ncgri-MuiPaper-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 css-vxrdkt-MuiTypography-root"
              >
                Offering Terms
              </h4>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Investment Period
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    1 Month
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Dividend Yield
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    100%
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Interest Rate
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    100%
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Investment Structure
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    investment structure
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Gross IRR
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    0%
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Leverage
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    0
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Equity Multiple
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    0%
                  </h6>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  itemscope=""
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
                  >
                    Distribution Frequency
                  </h6>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
                  >
                    distribution frequency
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
